packages/react-instantsearch-core/src/connectors/connectVoiceSearch.ts

```diff
@@ -0,0 +1,74 @@
+import createConnector from '../core/createConnector';
+import voiceSearchHelper from '../lib/voiceSearchHelper';
+import { refineValue, getCurrentRefinementValue } from '../core/indexUtils';
+
+function getId() {
+  return 'query';
+}
+
+/* tslint:disable:variable-name */
```

Comment:
You can use `tslint:disable-next-line` to avoid re-enabling it on the next line.

```js
// tslint:disable-next-line:variable-name
```